### DISCUSSION TITLE
Had to ask for android permission to access internet?

### DISCUSSION THREAD

**Discussion by ne0rrmatrix** [CommunityToolkit/Maui#2023] [Q&A]
I had an issue with a new maui app I have been working on. I was at the stage of starting Release mode testing. I wanted to build it to see if it would run/behave as expected. When I launched app after build I looked at logcat and was seeing http 400 errors. After testing I determined that I needed to add `<uses-permission android:name="android.permission.INTERNET" />` 

I looked on the google Developer pages and it says this should not be required. I am running android 14 with Dotnet Maui 8.0.60. I was building in release mode. I checked and going back I noticed this was present back to 8.0.20. I did not test earlier. This does not affect any of my other apps. I have multiple apps that do not require this permission. It is a default install permission and users should not need to request access to the internet. Or am I wrong? Does anyone have any idea why on this one app I need to specify that permission?

**Comment by ghost** [comment]
I do not think you need to ask for internet permission. You can check if internet is available in MAUI app. @ne0rrmatrix if this is still relevant let me know, so we can discuss further.
